go.sum

```diff
@@ -219,8 +219,9 @@ github.com/containernetworking/plugins v0.8.6/go.mod h1:qnw5mN19D8fIwkqW7oHHYDHV
 github.com/containernetworking/plugins v0.9.1/go.mod h1:xP/idU2ldlzN6m4p5LmGiwRDjeJr6FLK6vuiUwoH7P8=
 github.com/containers/common v0.43.1 h1:zJYLII+NwHdcU5YLkMeseciRLMGk3wwIHWTzX/WofzU=
 github.com/containers/common v0.43.1/go.mod h1:BAoVyRYlxKZKAYpHcFMdrXlIZyzbJp9NwKTgadTd/Dg=
-github.com/containers/image/v5 v5.15.0 h1:NduhN20ptHNlf0uRny5iTJa2OodB9SLMEB4hKKbzBBs=
 github.com/containers/image/v5 v5.15.0/go.mod h1:gzdBcooi6AFdiqfzirUqv90hUyHyI0MMdaqKzACKr2s=
```

Comment:
Why is the github.com/containers/image/v5 v5.15.0/go.mod h1:gzdBcooi6AFdiqfzirUqv90hUyHyI0MMdaqKzACKr2s= line not removed?

Reply:
\*shrug\* the tools do it that way, this is a machine-generated and machine-maintained file. It’s fairly similar to many other entries in this file.

If you are interested, reading the formal documentation, or more likely implementation, of the code maintaining `go.sum` might reveal a reason.